### ISSUE TITLE
Further advantages of data augmentation on convolutional neural networks | chaser

### ISSUE THREAD

**Issue by ming71** [ming71/ming71.github.io#42]
https://ming71.github.io/Further-advantages-of-data-augmentation-on-convolutional-neural-networks.html 

 论文发布日期：2018 [ICANN] ¶1. Introduction   引用之前也是德国人的一篇文章观点，这里站队了,不会是一个组的相互引用吧。  现在的正则化方法不是必要的；而数据增强不能被简单认为是一种正则化方法，但也能起到同等的正则化作用。同时通过实验对比发现数据增强的正则化效果比采用单纯的正则化方法如dropout,weight decay(L2)效果好，加上其具有普适性且无需调参，
